packetbeat/protos/http/http.go

```diff
@@ -629,7 +629,7 @@ func (http *httpPlugin) collectHeaders(m *message) interface{} {
 
 func (http *httpPlugin) setBody(result common.MapStr, m *message) {
 	if m.sendBody && len(m.body) > 0 {
-		result["body"] = string(m.body)
+		result["body.content"] = string(m.body)
```

Comment:
Probably better user here `result.Put("body.content", string(m.body)` so it creates the correct object.

Reply:
Done. Although the rest of this file use the square bracket notation to set fields. I'm not adjusting the rest of this file.